AutomatedTesting/Gem/PythonTests/Physics/tests/heightfield/AxisAlignedBoxShape_ConfigurationWorks.py

```diff
@@ -0,0 +1,101 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
```

Comment:
Same here, this file doesn't belong under Physics, it belongs under Terrain.

Reply:
Changed and moved.